test/reservation_manager_test.rb

```diff
@@ -0,0 +1,122 @@
+require_relative 'test_helper'
+
+describe "ReservationManager class" do
+  before do 
+    @reservation_manager = Hotel::ReservationManager.new(20)
+  end
+  
+  describe "Initializer" do
+    it "is an instance of ReservationManager" do
+      expect(@reservation_manager).must_be_kind_of Hotel::ReservationManager
+    end
+
+    it "initializes correct number of rooms" do
+      expect(@reservation_manager.rooms.length).must_equal 20
+    end
+
+    it "populates valid room objects" do
+      expect(@reservation_manager.rooms[1]).must_be_kind_of Hotel::Room
+    end
```

Comment:
A better check here would be to loop through all of the rooms with individual assertions:

```suggestion
    it "populates valid room objects" do
      @reservation_manager.rooms.each do |room|
        expect(room).must_be_kind_of Hotel::Room
      end
    end
```